ckanext/qgov/common/resources/scheming_presets.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
was this from another project or ckan core?

Reply:
"resource_url_upload" is the preset name used by `ckanext-scheming`. `data_qld_resource_url_upload` is the preset name used by `ckanext-data-qld`. This snippet overrides both to add the resource URL validator.